### ISSUE TITLE
Do not allocate with `new` expression for Ptr

### ISSUE THREAD

**Issue by SomeRanDev** [SomeRanDev/reflaxe.CPP#34]
This should cause error. Allocation should require explicit API function call.

```haxe
class SomeClass {
    public function new() {}
 }
 
 function main() {
    final a: cxx.Ptr<SomeClass> = new SomeClass();
 }
 ```

**Comment by maihd** [comment]
That's right! Although `new` call in C++ sometime act like anti-pattern. But this issue should be a article in FAQs or wiki best practices.

**Comment by SomeRanDev** [comment]
I'll document it. Even give details in the error message 👌 
I don't know why I didn't do this from the start, unexpected `new` allocs are such a hassle. 💀

**Comment by maihd** [comment]
👍
